docs/topics/strings.md

```diff
@@ -154,6 +155,76 @@ ${'$'}_9.99
 """
 ```
 
+> To avoid `${'$'}` sequences in multiline strings, you can use Experimental [multi-dollar string interpolation feature](#multi-dollar-string-interpolation).
+>
+{style="note"}
+
+### Multi-dollar string interpolation
+
+> The multi-dollar string interpolation is [Experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained).
+> It may be changed at any time. We would appreciate your feedback in [YouTrack](https://youtrack.jetbrains.com/issue/KT-2425).
+>
+{style="warning"}
+
+Multiline strings in Kotlin don't support backslash escaping for literals. To escape dollar signs (`$`), 
+you need to use the `${'$'}` construct to prevent string interpolation.
+This approach can make the code harder to read, especially when the string contains multiple dollar signs.
+
+The multi-dollar string interpolation feature allows you to specify how many consecutive dollar signs are needed to trigger interpolation.
+This lets you treat dollar signs as literal characters in multiline strings.  
+For example:
+
+```kotlin
+val KClass<*>.jsonSchema : String
+    get() = $$"""
+    {
+      "$schema": "https://json-schema.org/draft/2020-12/schema",
+      "$id": "https://example.com/product.schema.json",
+      "$dynamicAnchor": "meta"
+      "title": "$${simpleName ?: qualifiedName ?: "unknown"}",
+      "type": "object"
+    }
+    """
+```
+
+Here, the use of `$$` before the multiline string means that only two consecutive dollar signs trigger string interpolation,
+while single dollar signs remain as literal characters.
+
+You can specify how many dollar signs trigger interpolation.
+For example, you can use three consecutive dollar signs `$$$` to trigger string interpolation:
+
+```kotlin
+val requestData =
+    $$$"""{
+      "currency": "$",
+      "enteredAmount": "42.45 $$",
+      "$$serviceField": "none",
+      "product": "$$$productName"
+    }
+    """
```

Comment:
I would show the result of this block of code, because maybe it's not clear that `$$serviceField` will stay with _two_ dollars in the string.